### PR TITLE
Changes required by Hummingbird

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: Sources/HummingbirdLambda/LambdaHandler.swift [hunk 1]
```diff
@@ -51,16 +51,7 @@ struct LambdaFunctionHandler<L: LambdaFunction>: LambdaHandler {
...
-        do {
-            response = try await self.responder.respond(to: request, context: requestContext)
-        } catch {
-            if let error = error as? HTTPResponseError {
-                response = error.response(allocator: context.allocator)
-            } else {
-                throw error
-            }
-        }
+        let response = try await self.responder.respond(to: request, context: requestContext)
```
Comment (Joannis) [inline Sources/HummingbirdLambda/LambdaHandler.swift:54]: Why wouldn't we need to catch these errors?

**Comment by Joannis** [comment]
Note: We need to add codeowners to this repo

File: Sources/HummingbirdLambda/LambdaHandler.swift (hunk 1 above)
Comment (adam-fowler) [reply Sources/HummingbirdLambda/LambdaHandler.swift:54]: Because Router already does it for you

**Comment by adam-fowler** [comment]
> Note: We need to add codeowners to this repo

Shall I add a CODEOWNERS to the project template?

File: Package.swift [hunk 2]
```diff
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", from: "1.0.0-alpha.1"),
         .package(url: "https://github.com/swift-server/swift-aws-lambda-events.git", from: "0.3.0"),
         .package(url: "https://github.com/swift-extras/swift-extras-base64.git", from: "1.0.0"),
-        .package(url: "https://github.com/hummingbird-project/hummingbird.git", from: "2.0.0-beta.7"),
+        .package(url: "https://github.com/hummingbird-project/hummingbird.git", branch: "main"),
```
Comment (Joannis) [inline Package.swift:19]: Needs to be updated

**Review by Joannis — APPROVED** [review]

**Pull request merged**